Terminal.Gui/ConsoleDrivers/ConsoleDriver.cs

```diff
@@ -175,11 +175,14 @@ public void AddRune (Rune rune)
 				// Normalize to Form C (Canonical Composition)
 				string normalized = combined.Normalize (NormalizationForm.FormC);
 
-				Contents [Row, Col - 1].Runes = new List<Rune> { (Rune)normalized [0] }; ;
+				List<Rune> runes = new List<Rune> ();
```

Comment:
Well @tig, that code does what you think it does. Of course in the `UpdateScreen` the code must reflect this change, but isn't yet ready.

Reply:
When I reviewed this, I thought the PR was marked as ready for review. I may have mis-read that. Sorry for the unneeded comment.

Reply:
No you didn't have mis-read that. I wanted to say that it could be for another PR. This was only to make the unit test pass. But I'll try to do at least in `NetDriver`. I see a problem with the `Console.Write(output)` by not normalizing `ǫ́`. The string builder has the the right string on it but the `Console.Write` doesn't render it on the screen. Can you check that please by doing some test, please? Thanks.

Reply:
@tig see what I mean. This should outputted `é́` but show `ę`. I really don't know why because the string builder has the right output.

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/041e8d10-0e29-48d5-be96-0d3403ac78ce)

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/5c80b993-3e97-46f4-b443-1c4a824b9d4d)

Reply:
This is because conhost does not know how to render combining marks that can't be normalized. As far as I can tell NO terminal does. 

Hence why i abandoned trying to make what you're working on work.

Reply:
Yes pasting in the Windows Terminal only the second paste well and the first is bad :-). At least the code is right and when terminal support it later it's already done.

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/517ba96f-4c8e-4f02-8a2a-fe6b743361d2)

Reply:
I may be wrong: https://github.com/microsoft/terminal/pull/12307

Using `Cascadia Code` in WT

```pwsh
echo "[e`u{0301}`u{0301}]"
```

Prints:
![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/57822977-cebf-4a03-887c-aee7879e9e91)

Note that not all fonts work. E.g. FiraCode NFM Retna:
![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/0200f71b-551a-4996-b998-cc28f3f056b8)

Reply:
Using Cascadia Code in WT

![image](https://github.com/gui-cs/Terminal.Gui/assets/13117724/49079001-0ad7-45d9-8634-1f35f5a6b1c7)